lib/manager/bundler/update.js

```diff
@@ -14,7 +14,7 @@ function updateDependency(currentFileContent, upgrade) {
       currentFileContent.split('"').length >
       currentFileContent.split("'").length
         ? '"'
-        : "'";
+        : /* istanbul ignore next */ "'";
```

Comment:
@rarkins do we need to test this?

Reply:
Probably, yes. I guess it means we need a fixture with single quotes!

Reply:
ok